dev/devicelab/lib/tasks/gallery.dart

```diff
@@ -25,14 +25,29 @@ TaskFunction createGalleryTransitionTest({bool semanticsEnabled = false}) {
   return GalleryTransitionTest(semanticsEnabled: semanticsEnabled);
 }
 
+TaskFunction createGalleryTransitionE2EBuildTest(
+  List<String> args, {
+  bool semanticsEnabled = false,
+  bool enableImpeller = false,
+}) {
+  return GalleryTransitionBuildTest(
+    args,
+    testFile: semanticsEnabled ? 'transitions_perf_e2e_with_semantics' : 'transitions_perf_e2e',
+    needFullTimeline: false,
+    timelineSummaryFile: 'e2e_perf_summary',
+    transitionDurationFile: null,
+    timelineTraceFile: null,
+    driverFile: 'transitions_perf_e2e_test',
+    enableImpeller: enableImpeller,
+  );
+}
+
 TaskFunction createGalleryTransitionE2ETest({
   bool semanticsEnabled = false,
   bool enableImpeller = false,
 }) {
   return GalleryTransitionTest(
-    testFile: semanticsEnabled
-        ? 'transitions_perf_e2e_with_semantics'
-        : 'transitions_perf_e2e',
+    testFile: semanticsEnabled ? 'transitions_perf_e2e_with_semantics' : 'transitions_perf_e2e',
```

Comment:
Why did the formatting change here?

Reply:
I guess I formatted locally with 120 length.. Restored back.

Reply:
The `flutter/flutter` repo doesn't use the Dart formatter for formatting, so you might want to make sure you have it turned off in your IDE when working there.